bool charging
bool fully_charged
float32 soc
float32 remaining_time_charge
float32 remaining_time_run
float32 remaining_distance
float32 maximum_traveling_distance
